src/main/java/seedu/address/storage/JsonAdaptedMeeting.java

```diff
@@ -0,0 +1,96 @@
+package seedu.address.storage;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Meeting;
+
+import javax.xml.namespace.QName;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START_TIME;
+
+/**
+ * Jackson-friendly version of {@link Meeting}.
+ */
+class JsonAdaptedMeeting {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Meeting's %s field is missing!";
+
+    private final String person;
+    private final String startTime;
+    private final String endTime;
+    private final String location;
+
+    /**
+     * Constructs a {@code JsonAdaptedMeeting} with the given meeting details.
+     */
+    @JsonCreator
+    public JsonAdaptedMeeting(@JsonProperty("person") String person, @JsonProperty("phone") String startTime,
+                             @JsonProperty("email") String endTime, @JsonProperty("role") String location) {
+        this.person = person;
+        this.startTime = startTime;
+        this.endTime = endTime;
+        this.location = location;
+    }
+
+    /**
+     * Converts a given {@code Meeting} into this class for Jackson use.
```

Comment:
Urm, do you perhaps again, mean JSON instead of Jackson?

Reply:
Elaborated